api/config.py

```diff
@@ -292,18 +292,21 @@ class Configuration(CoreConfiguration):
             "label": _("The primary languages represented in this library's collection"),
             "type": "list",
             "description": LANGUAGE_DESCRIPTION,
+            "optional": True,
         },
         {
             "key": SMALL_COLLECTION_LANGUAGES,
             "label": _("Other major languages represented in this library's collection"),
             "type": "list",
             "description": LANGUAGE_DESCRIPTION,
+            "optional": True,
         },
         {
             "key": TINY_COLLECTION_LANGUAGES,
             "label": _("Other languages in this library's collection"),
             "type": "list",
             "description": LANGUAGE_DESCRIPTION,
+            "optional": True,
```

Comment:
It looks like these changes were brought in from SIMPLY-1318 work. This code shouldn't be in this branch, but as long as I'm talking about it, I think SMALL_ and TINY_COLLECTION_LANGUAGES can be made optional, I think LARGE_COLLECTION_LANGUAGES is still required. Although we could make ['eng'] the default.

Reply:
Actually, looking at https://github.com/NYPL-Simplified/circulation/blob/master/api/lanes.py#L135, I think the right thing to do is to make all three of these optional, with no default. If no value is set, we'll estimate values based on the library's actual holdings at the time the lanes are created.